docs/hacking/install.rst

```diff
@@ -186,64 +171,139 @@ system that Docker can be installed on:
 
    .. code-block:: bash
 
-      docker logs nsqd
+      sudo docker logs nsqd
 
    For more on how to use Docker see the `Docker website`_.
 
 
 .. _Docker website: https://www.docker.com/
 
 
-Running h
----------
+Installing the gulp command
+---------------------------
+
+Install ``gulp-cli`` to get the ``gulp`` command:
 
-Run the following command to start the Hypothesis web service. If this is a
-fresh clone of the repository, this may take some time to run, as it will need
-to install the application dependencies and build the client assets.
+.. code-block:: bash
+
+    sudo npm install -g gulp-cli
 
-.. note::
-    Although it is strictly optional, we highly recommend that you install h
-    inside a Python "virtualenv". Please refer to the `virtualenv environment`_
-    section featured in the Hitchhiker's guide to Python for a comprehensive
-    introduction.
 
-.. _virtualenv environment: http://docs.python-guide.org/en/latest/dev/virtualenvs/
+Getting the h source code from GitHub
+-------------------------------------
+
+Use ``git`` to download the h source code:
+
+.. code-block:: bash
+
+    git clone https://github.com/hypothesis/h.git
+
+This will download the code into an ``h`` directory in your current working
+directory.
+
+Change into the ``h`` directory from the remainder of the installation
+process:
+
+.. code-block:: bash
+
+   cd h
+
+
+Creating a Python virtual environment
```

Comment:
This is another case where I really can't endorse us attempting to teach people how the entire Python ecosystem works. If you want to link to http://docs.python-guide.org/en/latest/dev/virtualenvs/ or something, that would be great. But I *really* don't want us to have to maintain a set of documentation on how to use virtualenv.